languages.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
If we're defining languages here we should preferably reuse this definition for the submit script

Reply:
Can you elaborate?